.github/README.md

```diff
@@ -0,0 +1,50 @@
+# Hedera Smart Contracts Workflows
+
+A detailed overview of how you can use hedera-smart-contract's github workflows for testing purposes. Besides examples, there will be explanations of what is happening under the hood.
+
+## Actions
+
+### Migration Testing
+
+When there is a new version released, no matter if it is network-node, mirror-node, or relay, we want to make sure the entire ecosystem is working as expected and there are not any critical regressions introduced. Another concern with switching the network node's versions is "the migration" that is running under the hood. It occurs every time a node has been started and detects if there is data for migration. The migration testing workflow offers this type of testing where we can choose versions for pre/post image tags.
+
+Available workflow inputs are:
+- **initialNetworkNodeTag** - Specify the initial Network Node image tag
+- **initialMirrorNodeTag** - Specify the initial Mirror Node image tag
+- **initialRelayTag** - Specify the initial Relay image tag
+- **targetNetworkNodeTag** - Specify target Network Node image tag
+- **targetMirrorNodeTag** - Specify the target Mirror Node image tag
+- **targetRelayTag** - Specify the target Relay image tag
+- **preMigrationTestTags** - Specify the pre-migration test tags. Default: @pre-migration. It could be every tag we want (e.g. **@OZERC20**)
+- **postMigrationTestTags** - Specify the post-migration test tags. Default: @post-migration. It could be every tag we want (e.g. **@OZERC20**)
+
+Examples:
+- if we want to test possible state corruption between mono and mod versions, we could use a setup like this:
+    - initialNetworkNodeTag: 0.48.1
```

Comment:
Maybe add a comment-type explaination saying this is the last mono image tag, or something like that
``` suggestion
    - initialNetworkNodeTag: 0.48.1 # last tag for mono module
```